recipe/session/models/models.go

```diff
@@ -0,0 +1,134 @@
+package models
+
+import (
+	"net/http"
+
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+type HandshakeInfo struct {
+	JWTSigningPublicKey            string
+	AntiCsrf                       string
+	AccessTokenBlacklistingEnabled bool
+	JWTSigningPublicKeyExpiryTime  uint64
+	AccessTokenValidity            uint64
+	RefreshTokenValidity           uint64
+	SigningKeyLastUpdated          uint64
+}
+
+type CreateOrRefreshAPIResponse struct {
+	Session        SessionStruct                   `json:"session"`
+	AccessToken    CreateOrRefreshAPIResponseToken `json:"accessToken"`
+	RefreshToken   CreateOrRefreshAPIResponseToken `json:"refreshToken"`
+	IDRefreshToken CreateOrRefreshAPIResponseToken `json:"idRefreshToken"`
+	AntiCsrfToken  *string                         `json:"antiCsrfToken"`
+}
+
+type SessionStruct struct {
+	Handle        string      `json:"handle"`
+	UserID        string      `json:"userId"`
+	UserDataInJWT interface{} `json:"userDataInJWT"`
+}
+
+type CreateOrRefreshAPIResponseToken struct {
+	Token       string `json:"token"`
+	Expiry      uint64 `json:"expiry"`
+	CreatedTime uint64 `json:"createdTime"`
+}
+
+type GetSessionResponse struct {
+	Session     SessionStruct                    `json:"session"`
+	AccessToken CreateOrRefreshAPIResponseToken `json:"accessToken"`
```

Comment:
We had discussed to make this a pointer?

Reply:
cant make this a pointer because json cannot read pointer